src/masternodes/validation.cpp

```diff
@@ -2444,6 +2450,7 @@ static void ProcessEVMQueue(const CBlock &block, const CBlockIndex *pindex, CCus
     if (pindex->nHeight >= chainparams.GetConsensus().ChangiIntermediateHeight4) {
         cache.AddBalance(Params().GetConsensus().burnAddress, {DCT_ID{}, static_cast<CAmount>(blockResult.total_burnt_fees)});
         cache.AddBalance(minerAddress, {DCT_ID{}, static_cast<CAmount>(blockResult.total_priority_fees)});
+        assert(blockResult.total_burnt_fees + blockResult.total_priority_fees <= totalEvmFees);
```

Comment:
We should do a stricter check, since the total burnt fees and total priority fees should tally to the total evm fees of that evm block.

Reply:
Resolved in 74e07a2